### PR TITLE
adapt to DNF 2.0 API changes

### PR DESCRIPTION
**Pull request by ignatenkobrain**
References: https://bugzilla.redhat.com/show_bug.cgi?id=1380566
Signed-off-by: Igor Gnatenko <i.gnatenko.brain@gmail.com>

### CODE REVIEW

**Comment by ignatenkobrain** [comment]
Note that I didn't have time to test it.

**Comment by InStallker** [comment]
Can one of the admins verify this patch?

File: src/sbin/lorax [hunk 1]
```diff
@@ -182,7 +182,7 @@ def get_dnf_base_object(installroot, sources, mirrorlists=None, repos=None,
             os.mkdir(logdir)
 
     dnfbase = dnf.Base()
-    conf = dnfbase.conf
+    conf = dnfbase._conf
```
Comment (vpodzime) [inline src/sbin/lorax]: This doesn't look like a right thing to do.

Comment (ignatenkobrain) [inline src/sbin/lorax]: this was never part of API, so it got underscore in 2.0 release.

Comment (ignatenkobrain) [inline src/sbin/lorax]: hmm, actually looks like it's our bug. I will fix it

**Comment by ignatenkobrain** [comment]
@vpodzime fixed.

**Comment by vpodzime** [comment]
jenkins, this is ok to test

**Pull request merged**